life_expectancy/cleaning.py

```diff
@@ -0,0 +1,66 @@
+"""
+Assignment 1
+"""
+import argparse
+import numpy as np
+import pandas as pd
+
+def load_data():
+    """
+    load data from file
+    """
+    data = pd.read_csv('life_expectancy/data/eu_life_expectancy_raw.tsv', sep='\t')
+    return data
+
+def save_data(data):
+    """
+    save data to file
+    """
+    data.to_csv('life_expectancy/data/pt_life_expectancy.csv', index = False)
+
+def clean_data(data, country = 'PT'):
```

Comment:
Can you add more type hints?
```suggestion
def clean_data(data: pd.DataFrame, country: str = 'PT') -> pd.DataFrame:
```